tinkerpop/src/main/java/uk/gov/gchq/gaffer/gafferpop/GafferPopGraph.java

```diff
@@ -0,0 +1,759 @@
+     *
+     * @param id        vertex ID or edge ID to be queried for.
+     *                  You can use {@link Vertex}, {@link GafferPopEdge},
+     *                  EdgeId or just a vertex ID value.
+     * @param direction {@link Direction} of edges to return.
+     * @param view      labels of edges. Alternatively you can supply a Gaffer View serialised into JSON.
+     * @return iterator of {@link GafferPopEdge}
+     */
+    public Iterator<GafferPopEdge> edgesWithView(final Object id, final Direction direction, final View view) {
+        return edgesWithView(Collections.singletonList(id), direction, view);
+    }
+
+    /**
+     * This performs a getRelatedEdges operation on Gaffer.
+     *
+     * @param ids       vertex IDs and edge IDs to be queried for.
+     *                  You can use {@link Vertex}s, {@link GafferPopEdge}s,
+     *                  EdgeIds or just vertex ID values.
+     * @param direction {@link Direction} of edges to return.
+     * @param view      a Gaffer {@link View} containing edge groups.
+     * @return iterator of {@link GafferPopEdge}
+     */
+    public Iterator<GafferPopEdge> edgesWithView(final Iterable<Object> ids, final Direction direction, final View view) {
+        return edgesWithSeedsAndView(getElementSeeds(ids), direction, view);
+    }
+
+    @Override
+    public <C extends GraphComputer> C compute(final Class<C> graphComputerClass) throws IllegalArgumentException {
+        throw Exceptions.graphComputerNotSupported();
+    }
+
+    @Override
+    public GraphComputer compute() throws IllegalArgumentException {
+        throw Exceptions.graphComputerNotSupported();
+    }
+
+    @Override
+    public Transaction tx() {
+        throw Exceptions.transactionsNotSupported();
+    }
+
+    @Override
+    public Variables variables() {
+        return this.variables;
+    }
+
+    @Override
+    public Configuration configuration() {
+        return configuration;
+    }
+
+    @Override
+    public void close() throws Exception {
+        serviceRegistry.close();
+    }
+
+    @Override
+    public ServiceRegistry getServiceRegistry() {
+        return serviceRegistry;
+    }
+
+    @Override
+    public Features features() {
+        return features;
+    }
+
+    public <T> T execute(final OperationChain<T> opChain) {
+        for (final Operation operation : opChain.getOperations()) {
+            operation.setOptions(opOptions);
+
+            if (operation instanceof Input) {
+                Object input = ((Input) operation).getInput();
+                if (input != null) {
+                    if (input instanceof MappedIterable) {
+                        ((MappedIterable) input).forEach(item -> { LOGGER.info(item.toString()); });
+                    } else {
+                        LOGGER.info(input.toString());
+                    }
+                }
+            }
+
+        }
+
+        try {
+            LOGGER.info(opChain.toString());
```

Comment:
Should these instances of `LOGGER` be removed? This looks like something from debugging as there is no message explaining why this is logged.

Reply:
This was mainly for my debugging to see what the opChain is, maybe we can make a ticket to have this enabled via config?

Reply:
In that case it should have a message along the lines of `I am printing a thing: the thing`. Otherwise there will be operations printed to console without context.